docs/development.md

```diff
@@ -70,35 +72,37 @@ the Application's configuration.
 
 #### Run Asylo Server
 
-The following command builds and runs an Oak Server instance that is running a
-specific Oak Application (this version of the server is based on the
-[Asylo](https://github.com/google/asylo) framework):
+The following command builds and runs an Oak Server instance inside Docker,
+running a specific Oak Application. This version of the server is based on the
+[Asylo](https://github.com/google/asylo) framework, and uses an Asylo-specific
+toolchain to build the Oak Server.
 
 ```bash
-./scripts/run_server -s asylo -a "${PWD}/bazel-client-bin/examples/hello_world/config/config.bin"
+./scripts/docker_run ./scripts/run_server -s asylo -e hello_world
```

Comment:
The enumeration of the steps (line 55) already implies that they are to be executed in the context of the Docker container. In fact I think the instructions should just start with something like "run docker_sh, and then all the other commands inside that shell".